pagopa/pn-frontend
# PN-3842 fix: adjusts clickable areas for checkboxes in Recipient

## Short description
Previously the addresses checkbox clickable areas overlapped in between them, clicking on the lower area of the top one activated the bottom one, after this changes they should not overlap anymore and the clickable zone of each one should be more clear.

## List of changes proposed in this pull request
-  Adjusts clickable areas for checkboxes in Recipient

## How to test
Click on the lower area of the link 'Aggiungi un domicilio digitale', it should add the relative text field correctly.
